src/main/java/frc/trigon/robot/subsystems/intake/Intake.java

```diff
@@ -35,6 +35,19 @@ public IntakeConstants.IntakeState getTargetState() {
         return targetState;
     }
 
+    public boolean hasNote() {
+        return IntakeConstants.HAS_NOTE_BOOLEAN_EVENT.getAsBoolean();
+    }
+
+    /**
+     * Checks if a note has been collected early using the motor's current. This is quicker than `hasNote()` since it updates from the change in current (which happens right when we hit the note), instead of the distance sensor which is positioned later on the system.     *
```

Comment:
```suggestion
     * Checks if a note has been collected early using the motor's current. This is quicker than `hasNote()` since it updates from the change in current (which happens right when we hit the note), instead of the distance sensor which is positioned later on the system.
```